_health-care/_js/_form.js

```diff
@@ -1241,6 +1285,9 @@ var HealthApp = (function() {
           e.preventDefault();
           // TODO(awong): Remove this debugging logic when things work.
           console.log(build1010ezXml(formRoot));
+          for (var i = 0; i < allInputs.length; i++) {
+            validate(allInputs[i]);
```

Comment:
Probably want to create an event listener on the "Continue" button for each section. The `change` handler is on the whole form right now, so we probably don't want to watch for changes on the whole form when we really just care about a section at a time.